doc/user-guide/How-to-build.md

```diff
@@ -68,13 +68,10 @@ $ brew install erlang openssl unixodbc
 Add OpenSSL paths to the compiler and linker environment variables:
 
 ```bash
-$ export LDFLAGS="-L/usr/local/Cellar/openssl/[installed version]/lib/ -undefined dynamic_lookup $LDFLAGS"
-$ export CFLAGS="-I/usr/local/Cellar/openssl/[installed version]/include/ $CFLAGS"
+$ export LDFLAGS="-L/usr/local/opt/openssl/lib"
+$ export CFLAGS="-I/usr/local/opt/openssl/include"
```

Comment:
I think't be good to preserve exiting `CFLGAS` env var. Current export overrides it.